src/commonTypes.ts

```diff
@@ -47,17 +49,20 @@ export type PageFile<FileType> = {
 
 export type PageParams = ParsedUrlQuery;
 
-export type PageObject = {
-  page: PageFile<NextPageFile>;
-  appFile: PageFile<NextAppFile>;
+export type RouteInfo = {
```

Comment:
This is matching the return type of `getRouteInfo`: https://github.com/toomuchdesign/next-page-tester/pull/120/files#diff-768042b7438a04ad39b577c36622fd2716eec2cd53c5d896700b4383e055acceR70